README.md

```diff
@@ -189,6 +189,17 @@ val customInputComponentStyle = DefaultLightStyle.inputComponentStyle(
 paymentFormStyle.paymentDetailsStyle.cardNumberStyle.inputStyle = customInputComponentStyle
 ```
 
+To disable the `cardHolderName` and `BillingFormAddress` in the Payment Form, pass null value to respective style property in `PaymentDetailsStyle`
```

Comment:
Let's go with:

To disable `cardHolderName` and `BillingFormAddress` in the Payment Form, pass a null value to each respective style property in `PaymentDetailsStyle`: